### PR TITLE
Add Link Class

### PR DESCRIPTION
**Pull request by thomasyu888**
SYNPY-64 Extract targetId from Links and follow.

### CODE REVIEW

File: synapseclient/client.py [hunk 1]
```diff
@@ -739,6 +739,12 @@ def _getWithEntityBundle(self, entityBundle, entity=None, **kwargs):
         ifcollision = kwargs.get('ifcollision', 'keep.both')
         submission = kwargs.get('submission', None)
 
+        #If Link, get target ID entity bundle
+        if entityBundle['entity']['concreteType'] == 'org.sagebionetworks.repo.model.Link':
+            targetId = entityBundle['entity']['linksTo']['targetId']
+            targetVersion = entityBundle['entity']['linksTo']['targetVersionNumber']
+            entityBundle = self._getEntityBundle(targetId, targetVersion)
```
Comment (larssono) [inline synapseclient/client.py]: How do you now delete or find meta information about a link?

Comment (thomasyu888) [inline synapseclient/client.py]: If you add a link to a file, shouldn't the annotations should be the same as that of the file its being linked to?  So the meta information about a link is just the meta information of the target file.

Comment (thomasyu888) [inline synapseclient/client.py]: Updated. Added Param followLink which defaults to true, but if false, syn.get returns meta information of Link

**Comment by cbare** [comment]
SYNPY-64 Extract targetId from Links and follow.

**Pull request merged**